Добавление etl pipeline для persons

swats-the-floran opened the pull request.

Commit:
переименовал requirements.txt

Commit:
Реализовал etl pipeline для genres

Попутно поправил типизацию и отрефакторил для подключения нескольких pipelines.

Commit:
удалил неиспользуемый импорт

Commit:
Реализовал etl pipeline для persons

Commit:
исправил часть замечаний из ревью

Commit:
добавил комментарий

zhovtanovskaya commented:
Ветка `feature-ap-3-persons` была создана из `feature-ap-2-genres`. Это неправильный подход. Нужно было вернуться к `main` и создать `feature-ap-3-persons` от `main`. Благодаря чему этот бранч не содержал бы ETL для жанров. Вот [статья](https://habr.com/ru/post/106912/) которая поможет разобраться что к чему.

zhovtanovskaya committed:
Задокументированы ожидания в PostgresPersonLoader.load_all()

Описано требование, чтобы `film_work.modfied` менялся при
изменении списка персон фильма.
